### PR TITLE
Add database migrations to move APICredential setup to Services

### PR DESCRIPTION
**Pull request by sergei-maertens**
Related to #610 

**Changes**

In 20c654be1b8de8e3e9376213fcd9bf451a6a0a09 the config was changed to
get a client from the zgw_consumers.Service model rather than using
vng_api_common.APICredential. However, existing configuration for
people upgrading was not automatically migrated.

This commit adds migrations to automatically migrate the credentials to
zgw_consumers.Service.

### CODE REVIEW

**Comment by sergei-maertens** [comment]
Yeah, good catch!

File: src/openzaak/selectielijst/migrations/0003_move_config_to_service_model.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Service.objects.create(
+        label=credentials.label if credentials else "VNG Selectielijst",
+        api_type=APITypes.orc,
+        api_root=config.api_root,
+        oas=f"{config.api_root}schema/openapi.yaml",
+        client_id=credentials.client_id if credentials else "",
+        secret=credentials.secret if credentials else "",
+        user_id=credentials.user_id if credentials else "",
+        user_representation=credentials.user_representation if credentials else "",
+        auth_type=AuthTypes.zgw if credentials else AuthTypes.no_auth,
```
Comment (joeribekker) [inline src/openzaak/selectielijst/migrations/0003_move_config_to_service_model.py:30]: This is always no-auth I think? Even if a different selectielijst is used theres no way to tell if it uses ZGW auth.

Comment (sergei-maertens) [reply src/openzaak/selectielijst/migrations/0003_move_config_to_service_model.py:30]: True, was trying to accommodate for any edge cases, but it makes more sense to just leave it `no_auth` and let the end-user amend if needed.

**Comment by sergei-maertens** [comment]
@joeribekker tests should be fixed, and migration changes made (`nrc` type and not assuming the auth scheme for selectielijst)
